portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -6,21 +6,68 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.FullEntity;
+import com.google.cloud.datastore.KeyFactory;
+
+import com.google.cloud.datastore.Query;
+import com.google.cloud.datastore.QueryResults;
+import com.google.cloud.datastore.StructuredQuery.OrderBy;
+import com.google.gson.Gson;
+import com.google.sps.data.Info;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
+
+
 @WebServlet("/form-handler")
 public class FormHandlerServlet extends HttpServlet {
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
-    // Get the value entered in the form.
-    String textValue = request.getParameter("text-input");
+    String text = Jsoup.clean(request.getParameter("text-input"), Whitelist.none());
+    long timestamp = System.currentTimeMillis();
+
+    System.out.println("You submitted: " + text);
+
+    Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
+    KeyFactory keyFactory = datastore.newKeyFactory().setKind("Task");
+    FullEntity taskEntity =
+        Entity.newBuilder(keyFactory.newKey())
+            .set("title", text)
+            .set("timestamp", timestamp)
+            .build();
+    datastore.put(taskEntity);
```

Comment:
No need to fix for this review, but typically for readability working in industry, we would create something like DatastoreService. The benefits are:
- It makes the server code easier to read: you'll just have a single line saying datastoreService.storeTask(entity), for more readability, you could create a FullEntityMapper too. e.g. a class that takes in the response, and returns the FullEntity.
- This is reusable if you ever need to do similar actions, and can share logic such as getting the default datastore instance.